components/drawer/overview.md

```diff
@@ -12,12 +12,19 @@ position: 0
 
 The <a href="https://www.telerik.com/blazor-ui/drawer" target="_blank">Blazor Drawer component</a> is a dismissible or permanently visible panel for [navigating]({%slug drawer-navigation%}) in responsive web applications. It allows switching the content of different sections of the page. You can customize its [templates]({%slug drawer-templates%}), [display]({%slug drawer-modes%}), behavior when [minimizing]({%slug drawer-mini-mode%}) and respond to [events]({%slug drawer-events%}).
 
-#### To use a Telerik Drawer for Blazor
+## Creating Blazor Drawer
```

Comment:
Clarify some things:
* Data should be `IEnumerable<T>`
* `SelectedItem` should be of type `T`
* Explain what is the "content" of the Drawer.
* `@ref` is not used to toggle the Drawer, but to obtain reference to the component instance. And that is used to toggle the Drawer.
* When you say "Set the MiniMode and Mode properties", the reader doesn't know what that is, and if they want to do it. Either explain, or skip. Maybe explain `Mode` and skip the `MiniMore` for now.